Models/Day_model.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum
+
+
+class DaysEnum(Enum):
```

Comment:
one class per module. the convention is modules are `snake_case`, classes `PascalCase`